addons/markers/XEH_postInit.sqf

```diff
@@ -18,3 +18,24 @@ GVAR(currentMarkerPosition) = [];
 GVAR(currentMarkerAngle) = 0;
 GVAR(currentMarkerColorConfigName) = "";
 GVAR(currentMarkerConfigName) = "";
+
+// set marker pos local on every computer (prevent markers visible for everyone)
+[QGVAR(applyMarkerPosLocal), {
+    params["_marker", "_pos"];
+    _marker setMarkerPosLocal _pos;
```

Comment:
All units will run marker's `FUNC(setMarkerJIP)` which will set marker color, dir, pos and shape. The pos would be from when it was originally placed, this code will update the data to have the moved position.